regparser/tree/xml_parser/appendices.py

```diff
@@ -302,6 +315,12 @@ def is_subhead(tag, text):
                 self.paragraph_with_marker(
                     text,
                     tree_utils.get_node_text_tags_preserved(child))
+            elif child.tag == 'SEQUENCE':
+                old_depth = self.depth
+                self.end_group()
+                self.depth = old_depth
+                # import pdb; pdb.set_trace();
+                self.process_sequence(child)
```

Comment:
It's a hack to make X handle that series of marked paragraphs inside an unmarked environment. It only works on the one instance in one notice in Reg X.